src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -862,6 +862,7 @@
   "com.minecolonies.coremod.leavingcolony": "Farewell, you're leaving %s's colony.",
   "com.minecolonies.coremod.enteringcolonynotify": "Beware, %s just entered %s.",
   "com.minecolonies.coremod.leavingcolonynotify": "Farewell, %s has just left %s.",
+  "com.minecolonies.coremod.colonysizechange": "%d chunks around %s have been claimed for your colony",
```

```suggestion
  "com.minecolonies.coremod.colonysizechange": "%d chunks around %s have been claimed for %s.",
```

Maybe the name of the colony here? It gets confusing if you're an officer in multiple colonies.

---

It should be preceded by the "John Doe has completed the %s" that would identify which colony.

---

Then 
```suggestion
  "com.minecolonies.coremod.colonysizechange": "%d chunks around %s have been claimed for your colony.
```

---

Then 
```suggestion
  "com.minecolonies.coremod.colonysizechange": "%d chunks around %s have been claimed for your colony.",
```